src/gui/editors/AutomationEditor.cpp

```diff
@@ -615,6 +667,36 @@ void AutomationEditor::mousePressEvent( QMouseEvent* mouseEvent )
 
 
 
+void AutomationEditor::mouseDoubleClickEvent(QMouseEvent * mouseEvent)
+{
+	if (!validPattern()) { return; }
+
+	// If we double clicked inside the AutomationEditor viewport
+	if (mouseEvent->y() > TOP_MARGIN && mouseEvent->x() >= VALUES_WIDTH)
+	{
+		timeMap::iterator clickedNode;
```

Comment:
Can clear two assignments by doing this (and reusing the bool later if change `fineTuneValue` argument to `outValue` instead).
```suggestion
		const bool isOutVal = m_editMode == DRAW_OUTVALUES;
		timeMap::iterator clickedNode = getNodeAt(mouseEvent->x(), mouseEvent->y(), isOutVal);
```